### PR TITLE
feat: leave room

### PR DESCRIPTION
**Pull request by hsglc**
A generic modal was implemented. User profile bar has been updated. Nickname change form has been implemented.

The status of the user leaving the room has been implemented, but when he wants to rejoin the room he left, he cannot set the groomingInfo state, so it remains in the loading state on the board screen. I think I'm missing a point. Do you have any suggestions?

### CODE REVIEW

File: gurubu-client/src/app/styles/room/grooming-board-profile.scss [hunk 1]
```diff
@@ -43,51 +43,77 @@
...
+  //   border: 1px solid $gray-400;
+  //   border-radius: $radius-small;
+  // }
 
-  &__nickname-input:focus {
-    outline: none;
-  }
+  // &__nickname-input:focus {
+  //   outline: none;
+  // }
```
Comment (mertcanaltin) [inline gurubu-client/src/app/styles/room/grooming-board-profile.scss:80]: this area cleared?

**Comment by armagandalkiran** [comment]
thank you for your contribution! 🤘 this mr is huge, i will try to review it as soon as possible.

**Review by aahmetcakir — APPROVED** [review]

**Review by armagandalkiran — APPROVED** [review]
Thank you!

**Comment by mertcanaltin** [comment]
thanks, it didn't happen because you didn't delete the lobby from localStorage, we'll be opening a mr that fixes this 🚀

**Comment by aahmetcakir** [comment]
thank you 🚀

**Review by mertcanaltin — APPROVED** [review]
lgtm

**Pull request merged**